apps/deploy-web/src/pages/alerts/contact-points/[id]/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { EditContactPointPage } from "@src/components/alerts/EditContactPointPage";
+import { featureFlagService } from "@src/services/feature-flag";
+
+export default EditContactPointPage;
+
+export const getServerSideProps = featureFlagService.showIfEnabled("alerts");
```

Comment:
non-blocking: this needs to be replaced with `routeProtector.showToRegisteredUserIfEnabled` after some of my todays changes. This can totally be done when working on the edit page in the next PR